packages/truffle/test/scenarios/migrations/errors.js

```diff
@@ -131,22 +131,23 @@ describe("migration errors", function() {
     }
   });
 
-  it("should error on insufficient funds correctly [ @geth ]", async function() {
+  it("should error on insufficient funds correctly [ @geth ]", async function () {
     this.timeout(70000);
 
     try {
-      await CommandRunner.run("migrate -f 6", config);
+      console.log("about to run a migration");
```

Comment:
I already approved, and just noticed this rogue console.log

Reply:
Oh whoops! Good catch!